internal/common/config/config.go

```diff
@@ -25,17 +25,66 @@ type Config struct {
 
 	// MaxCriticalConsecutiveTurns is the maximum consecutive turns an island can be in the critical state.
 	MaxCriticalConsecutiveTurns uint
+
+	// Wrapped foraging config
+	ForagingConfig ForagingConfig
+
+	// Wrapped disaster config
+	DisasterConfig DisasterConfig
+}
+
+// ForagingConfig captures foraging-specific config
+type ForagingConfig struct {
+	// Deer Hunting
+	MaxDeerPerHunt        uint    // Maximimum possible number of deer on a single hunt (regardless of number of participants)
+	IncrementalInputDecay float64 // Determines decay of incremental input cost of hunting more deer
+	BernoulliProb         float64 // `p` param in D variable (see README). Controls prob of catching a deer or not
+	ExponentialRate       float64 // `lambda` param in W variable (see README). Controls distribution of deer sizes.
+
+	// Deer Population
+	MaxDeerPopulation     uint    // Maximimum possible deer population. Reserved for post-MVP functionality
+	DeerGrowthCoefficient float64 // Scaling parameter used in the population model. Larger coeff => deer pop. regenerates faster
+
+	// TOOD: add other pertinent params here (for fishing etc)
+}
+
+// DisasterConfig captures disaster-specific config
+type DisasterConfig struct {
+	XBounds         [2]float64 // [min, max] x bounds of archipelago
+	YBounds         [2]float64 // [min, max] y bounds of arch.
+	GlobalProb      float64    // Bernoulli 'p' param. Chance of a disaster occurring
+	SpatialPDFType  string     // Set x,y prob. distribution of the disaster's epicentre (more post MVP)
+	MagnitudeLambda float64    // Exponential rate param for disaster magnitude
 }
 
 // GameConfig returns the configuration of the game.
 // (Made a function so it cannot be altered mid-game).
 func GameConfig() Config {
+	foragingConf := ForagingConfig{
+		MaxDeerPerHunt:        4,
+		IncrementalInputDecay: 0.8,
+		BernoulliProb:         0.95,
+		ExponentialRate:       1,
+
+		MaxDeerPopulation:     12,
+		DeerGrowthCoefficient: 0.4,
+	}
+	disasterConf := DisasterConfig{
+		XBounds:         [2]float64{0, 10},
```

Comment:
@Eirikalb  added new types for Coordinate in the `shared` package in #85 . Let's use that for all coordinates so that we have better type-checking? :D

Reply:
Sure - makes sense 👍